aip/0216.md

```diff
@@ -99,18 +99,20 @@ State enums **should** live inside the resource definition.
 **TL;DR:** Clearly communicate to users that state enums may receive new values
 in the future, and be conscientious about adding states to an existing enum.
 
-Adding states to an existing states enum _can_ break existing user code.
-Consider a state with only two values: `ACTIVE` and `DELETED`. A user may add
-code that checks `if state == ACTIVE`, and in the else cases simply assumes the
-resource is deleted. If the API later adds a new state for another purpose,
-that code will break.
+Even though adding states to an existing states enum _can_ break existing user
+code, adding states is not considered a breaking change. Consider a state with
+only two values: `ACTIVE` and `DELETED`. A user may add code that checks
+`if state == ACTIVE`, and in the else cases simply assumes the resource is
+deleted. If the API later adds a new state for another purpose, that code will
+break.
 
 We ultimately can not control this behavior, but API documentation **should**
 actively encourage users to code against state enums with the expectation that
 they may receive new values in the future.
 
 APIs **may** add new states to an existing State enum when appropriate, and
-adding a new state is considered a non-breaking change.
+adding a new state is _not_ considered a breaking change; i.e. adding a new
```

Comment:
This seems to be repetitive. Perhaps just "...adding a new state is _not_ considered a breaking change."

Reply:
Works for me--dropped everything after the semicolon (inclusive)